sample/redo/lib/main.dart

```diff
@@ -49,6 +51,15 @@ class _MyHomePageState extends State<MyHomePage> {
   final Random _random = Random.secure();
 
   _MyHomePageState() {
+    _firebaseMessaging.requestNotificationPermissions(
+      const IosNotificationSettings(sound: true, badge: true, alert: true, provisional: false),
```

Comment:
[fix?] Did we give up on the Android version?

Reply:
`requestNotificationPermissions` is a no-op on android.

Reply:
Does `IosNotificationSettings` work on Android?

Reply:
Yeah. The whole line is just a no-op. This is straight from their tutorial. Added a comment.